Doc/library/typing.rst

```diff
@@ -32,6 +32,11 @@ In the function ``greeting``, the argument ``name`` is expected to be of type
 :class:`str` and the return type :class:`str`. Subtypes are accepted as
 arguments.
 
+New features have been added to the typing module in each major version of
+Python. The `typing_extensions <https://pypi.org/project/typing-extensions/>`_
+package provides backports to all supported versions of Python 3 for almost
```

Comment:
I think your rephrasing is good since it avoids these problems. "All supported versions" is supposed to mean all Python 3 versions that are still supported, and "almost all features" is because there's some things that can't be backported (e.g., PEP 604, PEP 585).